dlpy/utils.py

```diff
@@ -1116,7 +1123,7 @@ def get_txt_annotation(local_path, coord_type, image_size = 416, label_files = N
 
 
 def create_object_detection_table(conn, data_path, coord_type, output,
```

Comment:
do you mind also changing the other function create_object_detection_table_no_xml. Thanks